ui-tests/__test__/flows/feedback-form/cancel-feedback-form.ts

```diff
@@ -0,0 +1,14 @@
+import { Page } from 'playwright/test';
+import { getSelector, waitForAnimationEnd } from '../../helpers';
+import testIds from '../../../../src/helper/test-ids';
+
+export const cancelFeedbackForm = async (page: Page, skipScreenshots?: boolean): Promise<void> => {
+  const cancelButton = page.locator(getSelector(testIds.feedbackForm.cancelButton));
+  expect(cancelButton).toBeDefined();
+  await cancelButton.click();
+  await waitForAnimationEnd(page);
+
```

Comment:
nit: should we also check for the feedback form to be dismissed after this action?

Reply:
We do this with the screenshot comparison.